core-plugin/testSrc/com/google/gct/idea/appengine/validation/MethodParameterTypeInspectionTest.java

```diff
@@ -24,6 +24,14 @@
  */
 public class MethodParameterTypeInspectionTest extends EndpointTestBase {
 
+  @Override
```

Comment:
So does that mean it's not part of their open API?  Perhaps we should file a bug against them, since it sounds like a core part of their testing framework.